pisek/config/task_config.py

```diff
@@ -507,11 +472,18 @@ class ProgramLimits(BaseEnv):
     _section: str = "limits"
 
     time_limit: float = Field(ge=0)  # [seconds]
-    clock_limit: float = Field(ge=0)  # [seconds]
+    clock_mul: float = Field(ge=0)  # [1]
+    clock_min: float = Field(ge=0)  # [seconds]
```

Comment:
Nit: Note that this still doesn't allow us to match CMS's behaviour exactly, as it computes the clock limit as `time_limit * 2 + 1`

Reply:
I know. But:
1) Your solutions shouldn't timeout on wall clock anyway.
2) I kind of find this solution more intuitive
3) Most of the times you should retest it on CMS anyway.

Reply:
Fair.